src/App.jsx

```diff
@@ -14,57 +16,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodSelected, setGoodSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {goodSelected === null ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {goodSelected} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setGoodSelected(null)}
+          />
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map(item => {
```

Comment:
You are using the index as a key for the elements inside the map function. According to the checklist, you should avoid using index as a key. It's better to use unique and stable keys for elements in a list. In this case, you can use the `item` itself as a key, because all items in the `goods` array are unique.